bin/MadGraph5_aMCatNLO/cards/production/13TeV/exo_diboson/Spin-1/Wprime_Wh_WhadhVV2l2nu/Wprime_Wh_WhadhVV2l2nu_narrow_M2000/.dat

```diff
@@ -0,0 +1 @@
+## Models for searches of diboson resonances\ndibosonResonanceModel.tar.gz
```

Comment:
this seems to be a spurious file, please remove

Reply:
File removed.